packages/main/src/modules/npm.ts

```diff
@@ -1,6 +1,33 @@
+import path from 'node:path';
+import { existsSync } from 'node:fs';
 import { run } from './bin';
 
-export async function install(path: string) {
-  const installCommand = run('npm', 'npm', { args: ['install', '--loglevel', 'error'], cwd: path });
+export async function install(path: string, packageName?: string) {
+  const installCommand = run('npm', 'npm', {
+    args: ['install', '--loglevel', 'error', ...(packageName ? ['--save', packageName] : [])],
+    cwd: path,
+  });
   await installCommand.wait();
 }
+
+export async function packageOutdated(_path: string, packageName: string) {
+  const normalizedPath = path.normalize(_path);
+  const nodeModulesPath = path.join(normalizedPath, 'node_modules');
+
+  if (!existsSync(nodeModulesPath)) {
+    // For projects without node_modules, install dependencies before checking for outdated packages
+    await install(_path);
+  }
+
+  try {
+    const npmOutdated = run('npm', 'npm', {
+      args: ['outdated', packageName, '--depth=0', '--json'],
+      cwd: _path,
+    });
+    // If the npm outdated commands returns "{}", the package is updated otherwise, it returns a JSON object with outdated package versions
+    await npmOutdated.waitFor(new RegExp(packageName), /\{\}/);
```

Comment:
I see several `UnhandledPromiseRejectionWarning`s on the logs when the package is outdated:

```
11:53:07.332 › [UtilityProcess] {
  "@dcl/sdk": [
    {
      "current": "7.5.7",
      "wanted": "7.6.0",
      "latest": "7.6.0",
      "dependent": "@dcl/quests-client",
      "location": "/Users/***/.decentraland/New Scene 11/node_modules/@dcl/sdk"
    },
    {
      "current": "7.5.7",
      "wanted": "7.6.0",
      "latest": "7.6.0",
      "dependent": "New Scene 11",
      "location": "/Users/***/.decentraland/New Scene 11/node_modules/@dcl/sdk"
    }
  ]
}
11:53:07.333 › [UtilityProcess] Exiting "npm outdated @dcl/sdk --depth=0 --json" with pid=17991 and exit code=1
(node:17015) UnhandledPromiseRejectionWarning: Error: Error: process "npm outdated @dcl/sdk --depth=0 --json" with pid=17991 exited with code=1
    at ForkUtilityProcess.<anonymous> (file:///Users/***/Code/editor-electron/packages/main/dist/index.js:290:9)
    at ForkUtilityProcess.emit (node:events:519:28)
    at ForkUtilityProcess.a.emit (node:electron/js2c/browser_init:2:69597)
(node:17015) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 32)
```

It looks like the `npm oudated` command has a non-zero exit code when a package is outdated, and that ends up in a promise rejection that is not being caught

If the command has an exit code=0 when the package is up to date and a non-zero exit code when a package is outdated, maybe we could just use the `.wait()` helper instead of the `.waitFor` regex helper